keylime/tenant.py

```diff
@@ -1360,6 +1360,15 @@ def do_show_runtime_policy(self, name: Optional[str]) -> None:  # pylint: disabl
         if response.status_code >= 400:
             raise UserError(response_json)
 
+    def do_list_runtime_policy(self) -> None:  # pylint: disable=unused-argument
```

Comment:
Instead of the pylint ignore, you can declare this a static method.

Reply:
Pylint ignore was not needed.